selfswab/models.py

```diff
@@ -43,7 +44,6 @@ class SelfSwabScreen(models.Model):
     gender = models.CharField(max_length=10, choices=GENDER_CHOICES)
     facility = models.CharField(max_length=255, blank=False)
     risk_type = models.CharField(max_length=10, choices=RISK_TYPES)
-    timestamp = models.DateTimeField(default=timezone.now)
```

Comment:
Do we not need a migration for this?

Reply:
It was a duplicate field, but I'll run makemigrations just to be sure

Reply:
`No changes detected`